decidim-pages/app/commands/decidim/pages/create_page.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+module Decidim
+  module Pages
+    # Command that gets called whenever a component's page has to be created. It
+    # usually happens as a callback when the component itself is created.
+    class CreatePage < Rectify::Command
+      def initialize(component)
+        @component = component
+      end
+
+      def call
+        @page = Page.create(
+          title: @component.name,
+          component: @component
+        )
+
+        return broadcast(:ok) if @page
```

Comment:
This is wrong, if `create` fails it returns the object with the errors, not nil. From the docs: `Creates an object (or multiple objects) and saves it to the database, if validations pass. The resulting object is returned whether the object was saved successfully to the database or not.`